server-phoenix/lib/helios_web/schema.ex

```diff
@@ -7,7 +7,7 @@ defmodule HeliosWeb.Schema do
   end
 
   def get_string_key(%{source: source} = res, key) do
-    %{res | state: :resolved, value: Map.get(source, key)}
+      %{res | state: :resolved, value: Map.get(source, key)}
```

Comment:
Did you mean to indent this line here? It looks like a format issue.

Reply:
I need to get in a habit of running `mix format`. The only problem is I'll have to run it for individual files or it will most likely change a bunch of them and cause merge conflicts with ones that Lina might be working on.

Reply:
The commit message here is `fix: remove type checking on default absinthe middleware` and this commit just indents this line (appears to be incorrect). Could we drop the commit entirely?